packages/config/lib/validation.ts

```diff
@@ -147,14 +148,14 @@ const isOneOf = (...values) => {
  * Validates whether the supplied set of cert information is valid
  * @returns {string|true} Returns `true` if the information set is valid. Returns an error message if it is not.
  */
-const isValidClientCertificatesSet = (_key, certsForUrls) => {
+const isValidClientCertificatesSet = (_key: string, certsForUrls: any[]) => {
```

Comment:
I think this is the type
https://github.com/cypress-io/cypress/blob/3b2388797326e0f1ec8f011372e1352117e001a4/cli/types/cypress.d.ts#L2599